### PR TITLE
Unpkg

### PR DESCRIPTION
**Pull request by onokumus**
I think unpkg should not be needed for compilation.

### CODE REVIEW

File: vite.config.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default defineConfig({
         customMedia: true,
       },
       targets: {
-        chrome: 131,
+        chrome: 131 << 16,
```
Comment (felix-bohlin) [inline vite.config.ts:28]: What are we solving by setting the target to that?

**Comment by felix-bohlin** [comment]
Hey @onokumus, super grateful for your contribution! ❤️ Looks great - it makes it much nicer! Also easier to work with offline! Just wondering about the one thing above there, otherwise it's good to go 😊

File: vite.config.ts (hunk 1 above)
Comment (onokumus) [reply vite.config.ts:28]: https://github.com/parcel-bundler/lightningcss/issues/773#issuecomment-2270330817

Comment (felix-bohlin) [reply vite.config.ts:28]: Huh, wow 🤯 👍

**Pull request merged**